### DISCUSSION TITLE
Webpack and polyfill problem when starting ./reach react

### DISCUSSION THREAD

**Discussion by AleksaGK** [reach-sh/reach-lang#1577] [General]
Hello, I'm working on a Reach project and I'm running into issues when running the ./reach react command. I've copied code from the Reach GitHub repository (specifically the RPS-9-Web example found at https://github.com/reach-sh/reach-lang/tree/master/examples/rps-9-web). Additionally, I've added the following lines of code instead of MyAlgoConnect:

---------------------------------------------------------------------------------------
import { loadStdlib } from "@reach-sh/stdlib";
import { ALGO_MakePeraConnect as MakePeraConnect } from "@reach-sh/stdlib";
import { PeraWalletConnect } from "@perawallet/connect";

const reach = loadStdlib(process.env);

reach.setWalletFallback(
  reach.walletFallback({
    providerEnv: "TestNet",
    WalletConnect: MakePeraConnect(PeraWalletConnect),
  })
);

---------------------------------------------------------------------------------------

When I run ./reach react, I'm receiving an error message that includes 22 error messages (see code on the bottom). I'm running Reach version 0.1.13. Without this code, that I have written above, it compiles successfully so somewhere there is the error.

Any assistance with resolving this issue would be greatly appreciated. Thank you!

### ERROR:
Failed to compile.

Module not found: Error: Can't resolve 'url' in '/app/node_modules/stream-http'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "url": require.resolve("url/") }'
        - install 'url'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "url": false }
WARNING in ../stdlib/node_modules/express/lib/view.js 72:13-25
Critical dependency: the request of a dependency is an expression

WARNING in ../stdlib/node_modules/on-finished/index.js 191:11-33
Module not found: Error: Can't resolve 'async_hooks' in '/stdlib/node_modules/on-finished'

WARNING in ../stdlib/node_modules/raw-body/index.js 270:11-33
Module not found: Error: Can't resolve 'async_hooks' in '/stdlib/node_modules/raw-body'

ERROR in ./node_modules/stream-http/index.js 5:10-24
Module not found: Error: Can't resolve 'url' in '/app/node_modules/stream-http'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "url": require.resolve("url/") }'
        - install 'url'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "url": false }

...

ERROR in ../stdlib/node_modules/serve-static/index.js 20:10-24
Module not found: Error: Can't resolve 'url' in '/stdlib/node_modules/serve-static'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "url": require.resolve("url/") }'
        - install 'url'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "url": false }

ERROR in ../stdlib/node_modules/wait-port/lib/wait-port.js 2:12-26
Module not found: Error: Can't resolve 'net' in '/stdlib/node_modules/wait-port/lib'

webpack compiled with 22 errors and 3 warnings

**Comment by itsgio99** [comment]
I get the same exact error, did you find out how to fix it?
